api/types/mount/mount.go

```diff
@@ -26,6 +26,7 @@ type Mount struct {
 	Source   string `json:",omitempty"`
 	Target   string `json:",omitempty"`
 	ReadOnly bool   `json:",omitempty"`
+	State    State  `json:",omitempty"`
```

Comment:
I wonder if `State` is the right term here.

Would something like `Consistency` work?